R/exercise.R

```diff
@@ -317,31 +311,34 @@ evaluate_exercise <- function(exercise, envir, evaluate_global_setup = FALSE) {
     }
   }
 
+  # Setup a temporary directory for rendering the exercise
+  exercise_dir <- tempfile(pattern = "lnr-ex")
+  dir.create(exercise_dir)
+  withr::local_dir(exercise_dir)
+  on.exit(unlink(exercise_dir), add = TRUE)
```

Comment:
I've since realized we can do the same thing with

```suggestion
  withr::local_dir(withr::local_tempdir(pattern = "lrn-ex"))
```

but because we'll need to use `exercise_dir` at some later point (actually in #539) we can do it in two lines

```suggestion
  exercise_dir <- withr::local_tempdir(pattern = "lrn-ex")
  withr::local_dir(exercise_dir)
```